src/Groups/pcgroup.jl

```diff
@@ -365,3 +364,87 @@ function pc_group(c::GAP_Collector)
   end
 end
 
+"""
+    letters(g::Union{PcGroupElem, SubPcGroupElem})
+
+Return the letters of `g` as a list of integers, each entry corresponding to
+a group generator.
+
+# Examples
+```jldoctest
+julia> c = collector(2, Int);
+
+julia> Oscar.set_relative_orders!(c, [2, 3])
+
+julia> Oscar.set_conjugate!(c, 2, 1, [2 => 2])
+
+julia> gg = pc_group(c)
```

Comment:
The interface for construction `collectors` is quite advanced and currently not even fully documented, so using it for an example is maybe not optimal. I suggest to use the small groups library to get a suitable example group instead, i.e. here we can just do this:
```suggestion
julia> gg = small_group(6,1)
```
and then I think the rest of the example stays unchanged.

Same remark applies below.

Reply:
Perfect, no worries! I have now updated the examples with this replacement.